### ISSUE TITLE
B005 false positive on methods that happen to be named `strip`

### ISSUE THREAD

**Issue by 0xallie** [PyCQA/flake8-bugbear#297]
```py
import pysdhstrip

stripped = pysdhstrip.strip("1\n00:00:00,000 --> 00:01:00,000\nALICE: Good morning.")
```

The above code incorrectly triggers B005:
```
B005: Using .strip() with multi-character strings is misleading the reader. It looks lik...if this is deliberate. Use .replace() or regular expressions to remove string fragments.
```

Python 3.10.7, flake8-bugbear 22.9.23

**Comment by FozzieHi** [comment]
As we don't know the type of the variable the `strip` method is being called on I'm not entirely sure how we could solve this for every case. However, in this specific case we could check the imports and not flag on any imported modules. Is this something that we want to add?

**Comment by FozzieHi** [comment]
@cooperlees What do you think we should do with this?

**Comment by cooperlees** [comment]
I feel this would be a wack-a-mole that we'd never get right. We maybe could maybe make it configurable. That is:

- Add config to allow users to add B005 libraries to exclude triggering B005 from using `.strip()`

We do have another check like this I believe ... We would need to update config docs + add tests showing this work as usual etc. etc. ... But I wouldn't might that and some defaults hard coded if there are well know modules using `.strip()`.

**Comment by FozzieHi** [comment]
I agree configuration would be a good idea for the most coverage of this without users `noqa`ing the check. However I still think it's a good idea to just ignore the `strip()` method on module imports, such as:
```python
import test

test.strip()
```

Of course this won't work for imports such as this, as `test_var` could be a string:
```python
from test import test_var

test_var.strip()
```

I think this would catch most cases without adding configuration?

**Comment by cooperlees** [comment]
I did miss this. Yes, this could be worth doing and should quiet it in some cases ... So it's welcome.
